ddp-workspace/projects/ddp-dsm-ui/src/app/services/dsm.service.ts

```diff
@@ -230,10 +230,10 @@ export class DSMService {
     if (fileFormat) {
       map.push({name: 'fileFormat', value: fileFormat});
     }
-    if (typeof splitOptions === 'boolean') {
-      map.push({name: 'splitOptions', value: splitOptions});
+    if (typeof humanReadable === 'boolean') {
```

Comment:
Can the "humanReadable" ever be another type than the boolean?

Reply:
no, but based on my reading, this was the best/safest way to undefined-safe the method